boot/bootutil/src/encrypted.c

```diff
@@ -438,8 +438,14 @@ boot_enc_decrypt(const uint8_t *buf, uint8_t *enckey)
     uint8_t counter[BOOTUTIL_CRYPTO_AES_CTR_BLOCK_SIZE];
     uint16_t len;
 #endif
+    const struct bootutil_key bootutil_enc_key = {0};
```

Comment:
const struct means that this local `bootutil_enc_key` will remain {0} after a call to boot_retrieve_private_key